src/config-loader.js

```diff
@@ -103,6 +114,13 @@ class ConfigLoader {
   getProjectCi() {
     return this._projectCi;
   }
+  /**
+   * Get Project Format
+   * @return {String}
+   */
+  getProjectFormat() {
+    return this._format;
```

Comment:
what value will this return creating a new project from scratch?

Reply:
added `this._format = '.' + config.format;` to constuctor as default